src/templates/WpExtensionPlugin.js

```diff
@@ -0,0 +1,60 @@
+import React from "react"
+import {Box, Flex, Heading} from '@chakra-ui/core'
+import Layout from "../components/layout"
+import Container from "../components/container"
+import { graphql } from 'gatsby'
+import PageTransition from "../components/page-transition"
+import { ParseHtml } from "../components/parse-html"
+import Breadcrumb from "../components/breadcrumb/breadcrumb"
+
+const WpExtensionPlugin = ({data}) => {
+
+    const { wpContentNode: { title, content, uri } } = data;
+    const crumbs = [
+      {
+        title: `Extensions`,
+        path: `/extensions`,
+      },
+      {
+        title: title,
+        path: uri,
+        isCurrentPage: true,
```

Comment:
should we use Reach Router `useLocation()` to make this dynamic? 

We can visit later, just feels like we should be able to match the current path with the breadcrumb path to determine, eh?